src/configurations/destinations/userpilot/schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "configSchema": {
     "$schema": "http://json-schema.org/draft-07/schema#",
-    "required": [],
+    "required": ["token", "sdkEndpoint", "apiKey", "apiEndpoint"],
```

Comment:
@yashasvibajpai Is this expected change ?

Reply:
Yes, to unblock the failing schemaValidator warning, we did this.